packages/googlechrome.vm/googlechrome.vm.nuspec

```
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>googlechrome.vm</id>
+    <version>0.0.0.20231210</version>
+    <authors>Mandiant</authors>
```

This field should contain the authors of the tool
```suggestion
    <authors>Google LLC.</authors>
```